builtin/logical/jwt/secret_tokens.go

```diff
@@ -0,0 +1,159 @@
+package jwt
+
+import (
+	"fmt"
+	"time"
+	"encoding/json"
+
+	jwt "github.com/dgrijalva/jwt-go"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+// SecretCertsType is the name used to identify this type
+const SecretTokensType = "token"
+
+func secretTokens(b *backend) *framework.Secret {
+	return &framework.Secret{
+		Type: SecretTokensType,
+		Fields: map[string]*framework.FieldSchema{
+			"token": &framework.FieldSchema{
+				Type: framework.TypeString,
+				Description: "The Signed JWT Token",
+			},
+		},
+
+		DefaultDuration: 720 * time.Hour,
```

Comment:
I'd move this into a `const` also this seems like a strange default. Maybe 24h instead?